tools/pipelines/templates/include-publish-npm-package-steps.yml

```diff
@@ -138,6 +138,51 @@ steps:
                   exit 1
                 fi
             else
+              feedId=$(grep -oP '_packaging/\K[^/]+(?=/npm/)' publish_log)
+
+              if [[ $release == "release" && $feedId == "build" ]]; then
+                packageVersion=$(grep -oP 'npm notice version:\s+\K[^\s]+' publish_log)
+                
+                if [ "${{ parameters.artifactPath }}" == "scoped" ]; then
+                  scopes=("fluidframework" "fluid-tools" "fluid-internal" "fluid-experimental")
+                  for scope in "${scopes[@]}"; do
+                      if [[ "$packageName" == "$prefix"* ]]; then
+                          packageName="@${packageName/$scope-/$scope/}"
+                      fi
+                  done
+                fi
+
+                baseURL="${${{ parameters.feedName }}%/registry/}"
+                feedPromotionUrl="${baseURL/_packaging/_apis/packaging/feeds}"
+
+                response=$(curl -f -s -X PATCH \
+                  -H "Content-Type: application/json" \
+                  -H "Accept: application/json" \
+                  -u ":$(System.AccessToken)" \
+                  -d '{
+                        "views": {
+                          "op": "add",
+                          "path": "/views/-",
+                          "value": "Release"
+                        }
+                      }' \
+                      "$feedPromotionUrl"
+                  )
+                
+                curlExitStatus=$?
+                
+                if echo "$response" | grep -q '"success":"false"'; then
+                  reason="${response#*\"reason\":\"}"
```

Comment:
If this case plays out, what is the advised action? Rerun the pipeline? Manually promote?